### DISCUSSION TITLE
parked domain example.com

### DISCUSSION THREAD

**Discussion by AlbaniaOnline** [community/community#41771] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I cant add my website from Github to an parked domain, there DNS ip needed!
My website not show any IP of my page, and I cant add my website just by link.
I have try to get ip of my webpage, but Github does not show any IP for my website.

Please any help??

**Comment by airtower-luna** [comment, marked as answer]
How to configure your domain is described in the documentation: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site

It tells you which A, AAAA, or CNAME records to set up.
